packages/terra-form-field/src/Field.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconError from 'terra-icon/lib/icon/IconError';
+import 'terra-base/lib/baseStyles';
+import styles from './Field.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The form control elements the Field contains.
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid.
+   */
+  error: PropTypes.node,
+  /**
+   * Error Icon to display when the input is invalid.
+   */
+  errorIcon: PropTypes.element,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
+  /**
+   * Whether or not to hide the required indicator on the label.
+   */
+  hideRequired: PropTypes.bool,
+  /**
+   * The htmlFor attribute on the field label.
+   */
+  htmlFor: PropTypes.string,
+  /**
+   * Whether or not the field is invalid.
+   */
+  isInvalid: PropTypes.bool,
+  /**
+   * Whether or not the field is an inline field.
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Whether or not the label is visible. Use this props to hide a label while still creating it on the DOM for accessibility.
+   */
+  isLabelHidden: PropTypes.bool,
+  /**
+   * The label of the form control children.
+   */
+  label: PropTypes.node.isRequired,
+  /**
+   * Attributes to attach to the label.
+   */
+  labelAttrs: PropTypes.object,
+  /**
+   * Whether or not the field is required.
+   */
+  required: PropTypes.bool,
+  /**
+   * Whether or not to append the 'optional' label to a non-required field label.
+   */
+  showOptional: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  error: null,
+  errorIcon: <IconError />,
+  help: null,
+  hideRequired: false,
+  htmlFor: undefined,
+  isInvalid: false,
+  isInline: false,
+  isLabelHidden: false,
+  label: null,
+  labelAttrs: {},
+  required: false,
+  showOptional: false,
+};
+
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
+    if (context.intl === undefined) {
+      return new Error('Please add locale prop to Base component to load translations');
+    }
+  },
+};
+
+const Field = (props, { intl }) => {
+  const {
+    children,
+    error,
+    errorIcon,
+    help,
+    hideRequired,
+    htmlFor,
+    isInvalid,
+    isInline,
+    isLabelHidden,
+    label,
+    labelAttrs,
+    required,
+    showOptional,
+    ...customProps
+  } = props;
+
+  const fieldClasses = cx([
+    'field',
+    { 'field-inline': isInline },
```

Comment:
I'm not seeing a `field-inline` class defined in the SCSS file. I'm guessing that class would have styles to set field to display as an inline field. Is this something we are working on at a later phase?

Reply:
Oh shoot. Originally this styling was incorporated with a [media query](https://github.com/cerner/terra-core/pull/979/commits/13b62562cf02115c4fc0a5dd2fdacbcf8aefbceb), but Neil said lets remove that until we are certain this is the behavior we wanted. I replaced the .inline styling.